src/components/dashboard/Dashboard.js

```diff
@@ -23,7 +23,58 @@ import styles from '../../styles/DashboardStyles';
 // From Material UI: Container, Paper, Button, Typography 
 /* -------------------------------------------------------------------------- */
 
-//TODO The side panel should be abstracted into it's own component... I'm just realizing that. 
+
+// * Dummy data to be passed to the DecksView component - state will be held in Dashboard once the API is ready 
+const decks = [
```

Comment:
Starting to think about how state will be handled in-app. Right now Dashboard is concerned with the most.